shared/runtime/pyexec.h

```diff
@@ -37,6 +37,23 @@ extern pyexec_mode_kind_t pyexec_mode_kind;
 
 #define PYEXEC_FORCED_EXIT (0x100)
 
+#if MICROPY_PYEXEC_ENABLE_EXIT_CODE_HANDLING
+#define PYEXEC_NORMAL_EXIT (0)
+#define PYEXEC_UNHANDLED_EXCEPTION (1)
```

Comment:
How does one distinguish between an unhandled exception, and a `SystemExit(1)`?

Reply:
One doesn't. That is ok. It is not distinguishable in other programs as well. Only by additional stderr output